sdks/java/harness/build.gradle

```diff
@@ -55,21 +64,19 @@ description = "Apache Beam :: SDKs :: Java :: Harness"
 ext.summary = "This contains the SDK Fn Harness for Beam Java"
 
 dependencies {
-  dependOnShadedProjects.each {
-    implementation project(path: it, configuration: "shadow")
-  }
-  dependOnProjects.each {
-    implementation project(it)
-  }
-  shadow library.java.vendored_guava_26_0_jre
+  implementation project(path: ":model:fn-execution", configuration: "shadow")
+  implementation project(path: ":model:pipeline", configuration: "shadow")
+  implementation project(":runners:core-construction-java")
+  implementation project(":runners:core-java")
+  implementation project(":sdks:java:fn-execution")
```

Comment:
I would expect this to be provided

Reply:
Does this need a CHANGES.md update? Who should be providing it?

Reply:
Was wrong, we'll need to keep it as is.

Reply:
Reverted.